src/pykeen/sampling/negative_sampler.py

```diff
@@ -44,7 +57,39 @@ def num_entities(self) -> int:  # noqa: D401
         """The number of entities to sample from."""
         return self.triples_factory.num_entities
 
+    @property
+    def num_relations(self) -> int:  # noqa: D401
```

Comment:
this only gets used in one place. do you think it's worth making a property for that?

Reply:
From the perspective of code homogeneity it definitely is imo, since we also call the number of entities as a property of the negative sampler itself.